test/development/middleware-errors/index.test.ts

```diff
@@ -217,7 +217,7 @@ describe('middleware - development errors', () => {
               '\n    at <unknown> (middleware.js:3)' +
               // TODO: Should be ignore-listed
               '\n    at eval (middleware.js:3:12)' +
-              '\n    at (middleware)/./middleware.js (.next/server/middleware.js:40:1)' +
+              '\n    at (middleware)/./middleware.js (.next/server/middleware.js:18:1)' +
```

Comment:
Location in unsourcemapped stackframes doesn't matter so this is fine if it changes occasionally. The right fix will be to sourcemap these stacks.